lms/djangoapps/teams/models.py

```diff
@@ -100,6 +100,10 @@ def handle_activity(user, post, original_author_id=None):
         CourseTeamMembership.update_last_activity(user, post.commentable_id)
 
 
+def get_default_utc_date():
```

Comment:
Nit: naming similar to [Django docs](https://docs.djangoproject.com/en/1.11/ref/models/fields/#default)
```suggestion
def default_utc_date():
```